src/components/modal/addtostudio/presentation.jsx

```diff
@@ -11,6 +11,7 @@ const Spinner = require('../../spinner/spinner.jsx');
 const FlexRow = require('../../flex-row/flex-row.jsx');
 const StudioButton = require('./studio-button.jsx');
 const ModalTitle = require('../base/modal-title.jsx');
+const ModalInnerContent = require('../base/modal-inner-content.jsx');
```

Comment:
Curious about why it's specified as InnerContent, rather than just Content?

Reply:
To distinguish it from the existing "modal-content" css style (which I think isn't the right name), which wraps the entire modal.